src/functionalTest/resources/scenarios/RIA-6593-request-case-building.json

```diff
@@ -0,0 +1,48 @@
+{
+  "description": "RIA-6593 Request case building direction to legal representative",
```

Comment:
This functional test is testing whether the notification is sent out on the completion of event : requestCaseBuilding and not if the due date is appropriate for ADA case. Please have a look at : RIA-892-RIA-2694-request-respondent-evidence-start.json.